tox.ini

```diff
@@ -4,7 +4,29 @@ envlist = py36,py37,py38
 skipsdist = True
 
 [testenv]
+deps =
+    -r requirements-dev.txt
+    -e .
 commands =
-    pytest tests
-    mypy transcribe --ignore-missing-imports
-    black --check transcribe
+    pytest --cov-config=.coveragerc --cov=./ --cov-report=xml tests/functional tests/unit
+
+[testenv:lint]
+deps =
+    -r requirements-dev.txt
+commands =
+    black --check amazon_transcribe tests
+    # We ignore E203, E501 for this project due to black
+    flake8 --ignore=E203,E501 amazon_transcribe
```

Comment:
Let's move the ignores into a `[flake8]` section.

Reply:
Moved it into the setup.cfg so it's consistent across the project.